File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/user/UserProfileItemTest.java
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.user;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;

/**
 * Unit test class for {@link UserProfileItem}
 */
public class UserProfileItemTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(UserProfileItem.class)
				.verify();
	}
}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/user/UserProfileItemTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.user

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Unit test class for [UserProfileItem]
 */
class UserProfileItemTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(UserProfileItem::class.java)
				.verify()
	}
}